app/models/gems_adoption.rb

```diff
@@ -1,4 +1,9 @@
 class GemsAdoption < ActiveRecord::Base
   belongs_to :user
   validates :gem, :user_id, presence: true
+
+  def self.search(search)
+	  where("gem like ?", "%#{search}%") 
+	end
```

Comment:
I think this could be a scope, something like:
```ruby
scope :search, -> (keyword) { where("gem like ?", "%#{keyword}%") } 
```
What do you think?